target/linux/ath79/generic/base-files/etc/uci-defaults/04_led_migration

```diff
@@ -36,7 +36,8 @@ tplink,archer-c7-v2|\
 tplink,tl-wdr3600-v1|\
 tplink,tl-wdr4300-v1|\
 tplink,tl-wdr4300-v1-il|\
-tplink,tl-wdr4310-v1)
+tplink,tl-wdr4310-v1|\
+mercury,mw4530r-v1)
```

Comment:
The node should be sorted alphabetically internally, so mercury as the first member, and then the whole node should be moved based on sorting of the first entry.